SonarScanner.Shim/PropertiesWriter.cs

```diff
@@ -208,7 +208,13 @@ public void WriteSonarProjectInfo(string projectBaseDir, ICollection<string> sha
             AppendKeyValue(sb, SonarProperties.WorkingDirectory, Path.Combine(this.config.SonarOutputDir, ".sonar"));
             AppendKeyValue(sb, SonarProperties.ProjectBaseDir, projectBaseDir);
 
-            if (sharedFiles.Any())
+            this.projects.Select((p, index) =>
```

Comment:
I'm not a fan of these fairly complicated functional constructs, but it's rather a personal preference.